cmd/metal-api/internal/service/machine-service.go

```diff
@@ -1462,7 +1487,15 @@ func (r machineResource) finalizeAllocation(request *restful.Request, response *
 	}
 
 	old := *m
+
 	m.Allocation.ConsolePassword = requestPayload.ConsolePassword
+	m.Allocation.PrimaryDisk = requestPayload.PrimaryDisk
```

Comment:
What happens for already allocated machines which have been allocated before this feature was implemented ?